File: C/1071.c
#include <stdio.h>
 
int main() {
 
    int a,b;
    scanf("%d", &a);
    scanf("%d", &b);
    int soma = 0;
    if(a <= b){
        for(int i = a+1; i < b; i++){
            if(i%2){
                soma = soma + i;
            }
        }
    }else{
        for(int i = b+1; i < a; i++){
            if(i%2){
                soma = soma + i;
            }
        }
    }
    printf("%d\n", soma);
}

File: Python/1071.py
a = int(input())
b = int(input())
soma = 0
if a > b:
    if b < 0:
        b+=1
    for i in range(b, a):
        if i % 2 != 0:
            soma += i
            
else:
    if a < 0:
        a+=1
    for i in range(a, b):
        if i % 2 != 0:
            soma += i


print(soma)